doc/man7/password-handling.pod

```diff
@@ -0,0 +1,89 @@
+=pod
+
+=head1 NAME
+
+password handling - How diverse parts of OpenSSL treat pass phrases
+
+=head1 DESCRIPTION
+
+=head2 The general case
```

Comment:
I would move this head2 down to line 14 and leave the first paragraph (lines 11,12) as intro.

Reply:
Goo point.  Fixing